node/src/main/scala/com/wavesplatform/utils/RunNowExecutorService.scala

```diff
@@ -0,0 +1,13 @@
+package com.wavesplatform.utils
+
+import java.util
+import java.util.concurrent.{AbstractExecutorService, TimeUnit}
+
+object RunNowExecutorService extends AbstractExecutorService {
```

Comment:
Is this not the same as `MoreExecutors.newDirectExecutorService()`?

Reply:
Fixed